config/free_quotas.yml

```
@@ -0,0 +1,5 @@
+[{
+  name: inference-tokens,
+  value: 500000,
+  resource_type: "InferenceTokens"
+}]
```

We keep our YAML config list as single lines because it makes them easier to maintain when we have more items.

```suggestion
- { name: inference-tokens, value: 500000, resource_type: "InferenceTokens" }
```

---

Done.